### DISCUSSION TITLE
Tabbing out stopped all inputs from mouse and keyboard. Only ever happened to me with osu!

### DISCUSSION THREAD

**Discussion by Jessiewp** [ppy/osu#29605] [Q&A]
### Type

Game behaviour

### Bug description

Not sure if this was a windows issue or issue with osu but its never happened before but when i tabbed out of osu with alt tab my mouse and keyboard stopped working for no reason i couldnt press enter or anything and i had to restart my pc to fix it

### Version

2024.817.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16736130/compressed-logs.zip)

**Comment by Maulve** [comment]
I have a similar issue:

I'm on Arch Linux and when I tab out and back in, all keyboard presses (except search bar in map select) in the menu don't work. Mouse still works fine.
However, it still works in-game (mania).
